server/config/config.go

```diff
@@ -755,6 +758,7 @@ const (
 	defaultSchedulerMaxWaitingOperator = 5
 	defaultLeaderSchedulePolicy        = "count"
 	defaultStoreLimitMode              = "manual"
+	defaultEnableJointConsensus        = true
```

Comment:
will we start it by default?

Reply:
Yes, it improves stability. And this is on the `master` branch. If there are some bugs with this feature, we can turn it off on the `release` branch.